### ISSUE TITLE
[Request] Ability to limit and disable the nanobots

### ISSUE THREAD

**Issue by xenoflot** [ValkyrieofNight/EnvironmentalTech#19]
Hi,

On our server we permit true flight only with the RFTools Environmental Controller which has a range of effect. Outside the range of effect, the buff expires and you plummet to a messy end.

I can't see any options in ET to permit restriction or disabling machines. Would you please add configs to:

* Disable individual machines and modules (eg disable the solar system completely, yet leave the void miners operational, or disable just the flight module of the nanobot system)

* Adjust power requirements for modules

* Activate a limited range mode

* Specify that range

Thanks and Cheers!

**Comment by masterX244** [comment]
Usually you got minetweaker in your pack. that one allows to remove receipes of blocks so they get uncraftable. that should solve point 1

**Comment by xenoflot** [comment]
Indeed, we do have minetweaker and I have removed that recipe. Generally speaking though, I think it would be nice to have configs to prevent the machines/modules being registered in the first place. As your time and enthusiasm permits of course.

**Comment by xenoflot** [comment]
I didn't realise that the flight had no restriction and a player on my server has already created a flight module. I've since nerfed the recipe but I'd like to remove the flight module. 

I just visited his base and I am unable to break the modifier or main blocks of his nanobot controller. The structure blocks break and drop as expected, but the controller and modifier blocks are immediately replaced as if there's chunk protection. Is that something in ET?
